### PR TITLE
Use demo manifests to include resources

### PR DESCRIPTION
**Pull request by kusumachalasani**
While running the kruize local demo, we need to enable the resources to showcase different workload conditions.

### CODE REVIEW

**Review by chandrams — APPROVED** [review]
LGTM

File: monitoring/local_monitoring/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Local Monitoring with Kruize-Autotune
```
Comment (chandrams) [inline monitoring/local_monitoring/ReadMe.md:1]: You can remove autotune and just Kruize would do.

File: monitoring/local_monitoring/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+## Understanding the Demo
+
+This demo focuses on using the TFB (TechEmpower Framework Benchmarks) benchmark to simulate different load conditions and observe how Kruize-Autotune reacts with its recommendations. Here’s a breakdown of what happens during the demo:
+
+- TFB deployment in default Namespace
+    - The TFB benchmark is initially deployed in the default namespace, comprising two key deployments
+      - tfb-qrh: Serving as the application server.
+      - tfb-database: Database to the server.
+- Install Kruize
+  - Installs kruze under openshift-tuning name.
```
Comment (chandrams) [inline monitoring/local_monitoring/ReadMe.md:33]: kruze typo

File: monitoring/local_monitoring/ReadMe.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+```sh
+git clone git@github.com:kruize/kruize-demos.git
+```
+##### Change directory to the local monitoring demo:
+```sh
+cd kruize-demos/monitoring/local_monitoring
+```
+##### Execute the demo script in openshift as: 
+```sh
+./local_monitoring_demo.sh -c openshift
```
Comment (chandrams) [inline monitoring/local_monitoring/ReadMe.md:21]: Also update that the demo can be run on minikube as well with usage options

Comment (kusumachalasani) [reply monitoring/local_monitoring/ReadMe.md:21]: Didn't include as the minikube set-up doesn't run as expected for local monitoring.

File: monitoring/local_monitoring/ReadMe.md (hunk 1 above)
Comment (kusumachalasani) [reply monitoring/local_monitoring/ReadMe.md]: done!

File: monitoring/local_monitoring/ReadMe.md (hunk 2 above)
Comment (kusumachalasani) [reply monitoring/local_monitoring/ReadMe.md]: done!

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**